### PR TITLE
Xml comments for RootCommand

### PR DESCRIPTION
**Pull request by KathleenDollard**

### CODE REVIEW

File: src/System.CommandLine/RootCommand.cs [hunk 1]
```diff
@@ -7,12 +7,22 @@
 
 namespace System.CommandLine
 {
+    /// <summary>
+    /// A command representing entry point of an application and used to Invoke the correct handlers.
```
Comment (Keboo) [inline src/System.CommandLine/RootCommand.cs]: Not sure the Invoke potion of this is really accurate. This class is not actually responsible for doing the invocation, rather holding the data for parsing/invocation.

```suggestion
    /// A command representing entry point of an application.
```

Comment (jonsequitur) [reply src/System.CommandLine/RootCommand.cs]:
```suggestion
    /// A command representing an application entry point.
```

File: src/System.CommandLine/RootCommand.cs [hunk 2]
```diff
@@ -7,12 +7,22 @@
...
+    /// </summary>
     public class RootCommand : Command
     {
+        /// <summary>
+        /// Create a new instance of RootCommand
+        /// </summary>
+        /// <param name="description">The description of the command that could be used in a help display.</param>
```
Comment (jonsequitur) [inline src/System.CommandLine/RootCommand.cs]: Simpler?

```suggestion
        /// <param name="description">The description of the command shown in help.</param>
```

File: src/System.CommandLine/RootCommand.cs [hunk 3]
```diff
@@ -42,8 +52,14 @@ public override string Name
             Assembly.GetEntryAssembly() ??
             Assembly.GetExecutingAssembly();
 
+        /// <summary>
+        /// The name of the executable currently running, or if missing the first string argument.
```
Comment (jonsequitur) [inline src/System.CommandLine/RootCommand.cs]: > or if missing the first string argument.

I'm not sure I understand this bit.

Comment (KathleenDollard) [reply src/System.CommandLine/RootCommand.cs]: Looking at the code, that's what I thought it was doing.

**Comment by KathleenDollard** [comment]
Abandoning in favor of cleaner PR

**Pull request closed without merging**